datadog_cluster_agent/datadog_checks/datadog_cluster_agent/check.py

```diff
@@ -35,6 +35,7 @@
     'rate_limit_queries_period': 'datadog.rate_limit_queries.period',
     'rate_limit_queries_remaining': 'datadog.rate_limit_queries.remaining',
     'rate_limit_queries_reset': 'datadog.rate_limit_queries.reset',
+    'secret_backend.elapsed_ms': 'secret_backend.elapsed',
```

Comment:
i haven't tested it, but i think in this case you want `secret_backend__elapsed_ms` (double underscore as in line 14). this is how it's returned in prometheus (the internal check had different handling of double underscores)

Reply:
thanks
Indeed the metrics is `secret_backend__elapsed_ms`